kernelci/build.py

```diff
@@ -1286,6 +1286,21 @@ def _install_system_map(self, kbmeta, verbose):
             self._add_artifact('kernel', file_name, 'system_map')
             kbmeta['text_offset'] = '0x{:08x}'.format(text_offset)
 
+    def _install_vmlinux(self, kbmeta, verbose):
+        file_name = 'vmlinux'
+        vmlinux = os.path.join(self._output_path, file_name)
+        file_name_xz = 'vmlinux.xz'
+        vmlinux_xz = os.path.join(self._output_path, file_name_xz)
+        if os.path.exists(vmlinux):
+            if verbose:
+                print("Creating {}".format(file_name_xz))
+            shell_cmd("xz -T0 --compress < {src} > {dest}".format(
```

Comment:
Probably you can use same approach as we have already in code? (i know it ingest whole file in RAM, but i think it is fine with size we have):
https://github.com/kernelci/kernelci-core/blob/4c6c5065c4e9a0b57bbedc148db6edc517876764/kci_build#L373

Reply:
I can't tell what you're suggesting here, sorry.